.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "gradle"
```

Comment:
How about only for workflows? I feel like this is a repo that needs to dogfood itself for bumping versions :)

Reply:
This with the dogfood is right

Reply:
Ah I mean this plugin is to find dependencies that need to be updated. So we should probably use the plugin to find dependencies to update when releasing rather than using a workflow.

Reply:
To be honest, with dependabot this plugin is obsolete. Or at least much easier to handle

Reply:
> So we should probably use the plugin to find dependencies to update when releasing rather than using a workflow.

Feel free to make a pull request

Reply:
Left a suggested edit if making it only update workflows. Guess @ben-manes will decide if it's worth having the dogfooding or not :)

Reply:
For dogfooding, would it make more sense to use this action?
https://github.com/marketplace/actions/gradle-update-checker

Older versions of Gradle did not let plugins run on themselves, but better classloader isolation seems to let this work now.

Unfortunately, some users find dependabot too limiting because it uses regex scraping of build files. That can be problematic, e.g. see https://github.com/dependabot/dependabot-core/issues/1164 and a [tweet](https://twitter.com/CedricChampeau/status/1283361533351858182) by a Grade dev. There is also a lack of strictness regarding version conventions, e.g. some projects use an odd version to indicate an unstable preview. Hopefully over time better tooling and normalizing conventions will allow this plugin to be obsolete, but we aren't there yet.

Reply:
This pull request will work. It's up to you to close or approve.
Improvement can be done anytime

Reply:
okay. I am fine merging as is